### ISSUE TITLE
Can't build the docker-example

### ISSUE THREAD

**Issue by safepavel** [gruntwork-io/infrastructure-as-code-training#17]
I'm in macOS Catalina 10.15.6 running docker 19.03.8.

When I follow the instructional video and `docker build -t simpleserver .`, the first `RUN` command that installs `nodejs` and `npm` halts in a menu asking me to select my time zone interactively. Apparently docker builds aren't interactive, so this just hangs, and there's no way around it.

If I add `ARG DEBIAN_FRONTEND=noninteractive` near the beginning of the `Dockerfile`, then the first install appears to go fine, but I run up against an issue when I `RUN npm install -g nodemon`: "The command '/bin/sh -c npm install -g nodemon' returned a non-zero code: 1"

Slipping in to an ubuntu image with `docker run -it --rm ubuntu bash`, I can run those commands in sequence without a problem. (I get a different problem when I run `ln -s /usr/bin/nodejs /usr/bin/node`, because it tells me that symbolic link already exists, but that's beside the point.)

I've tried adding `ENV TZ=America/Denver` near the top of the `Dockerfile` in the hope this would allow the installer to just use it and not have to ask and not be confused by the absence of a timezone specification, but this was in vain.

tl;dr Something is broken.

**Comment by brikis98** [comment]
Thanks for reporting! Fix is here:  https://github.com/gruntwork-io/infrastructure-as-code-training/pull/18.

**Comment by brikis98** [comment]
OK, give the latest in `master` a shot and let me know if it works better!

**Comment by safepavel** [comment]
It works. As we say here: 🎉

**Comment by brikis98** [comment]
Great to hear!
